test/mocks/network/fetch.ts

```diff
@@ -157,13 +158,14 @@ const handleResponse = <R>(api: string, res: SoapResponse<R>): R | ErrorSoapBody
 				goToLogin();
 			}
 		}
-		console.warn(
+		// disable this part as it makes the tests fails
```

Comment:
I have a doubt about this commented part as the real implementation actually run this code as well. Should we really completely remove it or try to find a way to make the tests work knowing this parti will be executed too?

Reply:
Probably the cause of the failing tests are because of the jest configuration on your project.
In Mails we overcame this issue by setting the _jest-setup.ts_ like this:

`failOnConsole({
	...getFailOnConsoleDefaultConfig(),
	shouldFailOnWarn: false
});`

Reply:
yep! this is what I needed, thanks